tensorbay/client/dataset.py

```diff
@@ -764,7 +765,7 @@ def upload_segment(
             skip_uploaded_files: Set it to True to skip the uploaded files.
 
         Raises:
-            TypeError: When all the frames have the same patterns(both have frame id or not).
+            FrameError: When all the frames don't have the same patterns(both have frame id or not).
```

Comment:
better to not use abbreviations like `don't` in written language, use `do not` instead.